sql/mob_groups.sql

```diff
@@ -1044,7 +921,6 @@ INSERT INTO `mob_groups` VALUES (996,2412,37,0,128,4414,0,0,75,80,0);  -- Lights
 INSERT INTO `mob_groups` VALUES (997,2413,37,0,128,4416,7000,1000,75,80,0);  -- Light_Elemental      OK
 INSERT INTO `mob_groups` VALUES (998,2691,37,0,128,4411,9000,500,75,80,0);  -- Moblin_Dustman          OK
 INSERT INTO `mob_groups` VALUES (999,636,37,0,128,0,15000,0,75,80,0);  -- Mystic_Avatar (carbuncle)
-INSERT INTO `mob_groups` VALUES (1000,2836,37,0,128,0,0,0,75,80,0);  -- Nguruvilu
```

Comment:
was Nguruvilu a duplicate or what?

Reply:
Dunno, but Nguruvilu shouldn't have a group in zone 37 (Temenos).  It does already have a group in Abyssea - La Theine.
https://ffxiclopedia.fandom.com/wiki/Nguruvilu

Reply:
That's "yes" then.